mart/attack/adversary.py

```diff
@@ -110,13 +111,28 @@ def perturber(self) -> Perturber:
     def configure_optimizers(self):
         return self.optimizer(self.perturber)
 
+    def get_input_adv(self, *, input, target):
+        perturbation = self.perturber(input=input, target=target)
+        input_adv = self.composer(perturbation, input=input, target=target)
+        return input_adv
```

Comment:
This bothers me for some reason that I cannot quite yet pinpoint.

Reply:
I delete it in 1f43da1ff38cf4918310ca0abbfe66c0c20bc1ec